en/docs/integrations/integration-concepts.md

```diff
@@ -30,11 +28,11 @@ Choreo supports the following statements:
 
  - **Log**: Logs an event with an information statement or an error that occurs in your Choreo integration.
  - **Variable**: Declares a variable and a value of a preferred data type (`string`, `int`, etc.) to use later in the logical process of the Choreo integration.
- - **Data Mapping**: Allows to create variables by visually mapping one or more variables. For more information see [Data Mapping](../references/data-mapping.md).
+ - **Data Mapping**: Allows to create variables by visually mapping one or more variables. For more information, see [Data Mapping](../references/data-mapping.md).
  - **If**: Performs conditional execution.
  - **ForEach**: Includes a control flow statement in your Choreo integration to iterate over a list of items.
  - **While**: Continuously executes a block of statements in a loop based on a given `boolean` condition. 
- - **Respond**: Sends a specific response back to the client. For example, if the request was processed successfully,  responding to the client with an `HTTP_OK`.
+ - **Respond**: Sends a specific response back to the client. For example, if the request was processed successfully, responding to the client with an `HTTP_OK`.
```

Comment:
Shall we fix the same issues in service concepts page as well

Reply:
Pushed the changes after fixing the punctuation and spacing in service concepts page.